conanfile.py

```diff
@@ -195,6 +198,29 @@ def imports(self):
         self.copy("LICENSE*", dst="licenses", folder=True, ignore_case=True, excludes=excludes)
         self.copy("LICENCE*", dst="licenses", folder=True, ignore_case=True, excludes=excludes)
 
+        if not self.options.system_qt:
+            chromium_licenses = conan_helpers.gather_chromium_licenses(self.deps_cpp_info["qt"].rootpath)
+            chromium_licenses.sort(key=lambda license_info: license_info["name"].lower())
+
+            with open(os.path.join(self.install_folder, "NOTICE.Chromium.csv"), "w") as fd:
+                writer = csv.DictWriter(fd, fieldnames=["name", "url", "license"], extrasaction='ignore')
+                writer.writeheader()
+
+                for license in chromium_licenses:
+                    writer.writerow(license)
```

Comment:
Should we check the entries for commas? Quite unlikely, but it could happen in the url, and we could urlencode the string to be sure.

Reply:
We don't need to check for commas because the csv-writer takes care of proper escaping of the CSV fields.
And we probably also don't want to do URL-encoding since this field sometimes contains explanatory text why no URL is available, so it's not always a URL.

Reply:
Right, good point, I forgot that the csv writer should take care of this. Disregard my comment :-)